src/content/Settings/ExportImport/ExportImportIndividual/ExportImportIndividual.tsx

```diff
@@ -41,7 +38,10 @@ const ExportImportIndividual = ({
     event: React.ChangeEvent<HTMLInputElement>,
     worklistNumber: number
   ) => {
-    setImportInProgress(true)
+    dispatchModal({
```

Comment:
I thought we couldnt because they have specific selection logic but I just realized as I am typing this that the body of your modal layer can take JSX elements so it might be possible, i will look